docs/general/gotchas.md

```diff
@@ -21,3 +26,51 @@ import 'file?name=[name].[ext]!../img/yourimg.png';
 ```
 
 Then webpack will correctly transfer the image to the build folder.
+
+## Load reducers optimistically
+
+If you have containers that should be available throughout the app, like a `NavigationBar` (they aren't route specific), you need to add their respective reducers to the root reducer with the help of `combineReducers`.
+
+``` js
+// In app/reducers.js
+
+...
+import { combineReducers } from 'redux-immutable';
+...
+
+import navigationBarReducer from 'containers/NavigationBar/reducer';
+
+export default combineReducers({
+  route: routeReducer,
+  global: globalReducer,
+  language: languageProviderReducer,
+  navigationBar: navigationBarReducer,
+  ...asyncReducers,
+});
+```
+
+## Exclude modules from Babel processing
+
+You need to exclude node packages from babel processing which are not intended for browsers. Just add the package name to `exclude` array in `internals/config.js` and you're all set!
+
```

Comment:
It's in our [`config.js`](https://github.com/react-boilerplate/react-boilerplate/blob/master/internals/config.js) :bow: It's there in `exclude` because babel doesn't process css, maybe?